Makefile.toml

```diff
@@ -1355,3 +1355,11 @@ run_task = "newline-terminator"
 [tasks.check-git-diff]
 command = "git"
 args = ["diff", "--exit-code"]
+
+[tasks.map-js-token]
+script_runner = "@duckscript"
+script = '''
+file = readfile resources/web/map/js/trajectory_raw.js
```

Comment:
When you create a dist for the project, is it trivial to find the token string within the dist directory?  Can you do something like this

```
❯ rg 'pk[.].*'
resources/web/map/js/trajectory.js
5:mapboxgl.accessToken = "pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw";
```

Reply:
yea i think it should only be in that path since the required `trajectory.js` needs to be compiled before doing all the resources.qrc else it'll error not found.

Reply:
I don't think this'll be sufficient, we'll need to at least obfuscate this a bit (somehow)

```
swift-toolbox/py39-dist on  main took 6s 
❯ grep -nra 'pk[.]ey'
lib/python3.9/site-packages/swiftnav_console/console_resources.pyc:3994:mapboxgl.accessToken = "pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw";
```

Reply:
but i think if really wanted to, could find the key from outgoing traffic as well since the API is called as
`https://api.mapbox.com/styles/v1/mapbox/light-v11?access_token=pk.eyJ1Ijoic3dpZnQtYWRyaWFuIiwiYSI6ImNsZTN1MW82bDA2OGgzdXFvOWFuZTJlMnEifQ.9nR8m0C-B_ISNR4r4cMExw`

Reply:
It would be encrypted, so they'd have to break encryption on their machine and insert a MITM in order to extract the key which is much higher barrier than just grepping for a pattern that the key starts with.

Reply:
hmmm are we OK with still being able to do:
```
grep -nra 'accessToken'
```
and just running the decode function from `trajectory_raw.js`
🤔 
i guess its better than just grepping the entire key

Reply:
I think we've spent enough effort on this for the amount of benefit we're going to get.